src/Aspire.Hosting/Dashboard/DashboardViewModelService.cs

```diff
@@ -294,6 +325,82 @@ private async Task WriteChange(ResourceViewModel resourceViewModel, ObjectChange
             .ConfigureAwait(false);
     }
 
+    // Check a service endpoint
+    private async Task<bool> CheckServiceEndpointReady(string proxyUrlString, string errorContent)
+    {
+        using HttpClient client = new();
```

Comment:
Add a comment about explicitly creating a new client so make sure we use new tcp connections.